### PR TITLE
Added incentive calculation and transfer when model is improved

### PR DESCRIPTION
**Pull request by jsn5**
The incentive is calculated for the miners when they improve the model and it is transferred to them.

### CODE REVIEW

**Comment by iamtrask** [comment]
@jsn5, Can we change the notebook to demonstrate this working before we merge this PR?

File: contracts/ModelRepository.sol [hunk 1]
```diff
@@ -39,6 +39,12 @@ contract ModelRepository {
 
   }
 
+  function transfer(address reciever, uint amount){
+    if(!reciever.send(amount)){
+      throw;
+    }
+  }
```
Comment (anoff) [inline contracts/ModelRepository.sol]: shouldn't this implement an actual transfer of funds? 🤔

Comment (iamtrask) [reply contracts/ModelRepository.sol]: receiver.send(amount) transfers the funds.

Comment (anoff) [reply contracts/ModelRepository.sol]: ohh I thought it's just a check but makes sense now 🙇

**Comment by jsn5** [comment]
I've added the changes to the notebook to demo the incentive calculation and fund transfer. Please check PR to PySonar.

**Pull request merged**